kernel/init.c

```diff
@@ -191,6 +182,15 @@ void z_data_copy(void)
 
 bool z_sys_post_kernel;
 
+void immediate_printk(const char *fmt, ...)
```

Comment:
why are you adding it ? What it the problem with `printk` ? I presume is something related with `log_printk` ... but I don't know what it is.

Reply:
see above